kong/values.yaml

```diff
@@ -1,13 +1,15 @@
 Component: kong
 Name: kong
 Image: kong:0.13.0
+Registry: docker.io
```

Comment:
Why does the registry need to be specified separately -- is specifying docker.io required for the pull secret to match? If any changes are made here I'd prefer them to be in-line with our service charts, which specify "tag" and "repository".

Reply:
This is done to try to match the format with the public charts we use. Also Specifying it like this means changing the registry URL is simple to do if needed.

This is not yet at industry-standard since most use:
```
image:
          Registry: 
```

Reply:
This is done to easily change it and to get closer to matching how its done on public charts like bitnami.

Reply:
I had a look at a few prominent charts and none of them use registry. They mostly use image and tag

* https://helm.sh/docs/chart_best_practices/pods/#images separates as image and tag
* https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml separates as image and tag
* https://github.com/jenkinsci/helm-charts/tree/main/charts/jenkins uses image and tag